srcpkgs/gnuradio/template

```diff
@@ -1,32 +1,31 @@
 # Template file for 'gnuradio'
 pkgname=gnuradio
-version=3.7.13.5
-revision=2
+version=3.8.0.0
+revision=1
 build_style=cmake
-nocross="relies on volk, which is not complete for arm"
 conf_files="/etc/gnuradio/conf.d/*"
 configure_args="-DENABLE_INTERNAL_VOLK=OFF"
```

Comment:
gnuradio installs (by default) its python files in /usr/lib/python3.6/dist-packages/, so please add here:
`-DGR_PYTHON_DIR=${py3_sitelib}`